pyuvdata/uvdata/uvdata.py

```diff
@@ -4793,6 +4754,9 @@ def conjugate_bls(self, convention="ant1<ant2", *, use_enu=True, uvw_tol=0.0):
             )
             self.Nbls = np.unique(self.baseline_array).size
 
+            self.__antpair2ind_cache = {}
+            self._key2ind_cache = {}
```

Comment:
Why not just call the `_clear_antpair2ind_cache` method here?

Reply:
good idea...